app.R

```diff
@@ -13,7 +13,10 @@ library(tools)
 library(writexl)
 library(readxl)"
 
+# "masterText" collects text for an Rscript that will replicate commands executed by Good Nomen
```

Comment:
Thanks for adding comments. That will help make it clear to others (and to us) what the code is doing, especially for the trickier parts.